hadoop-ozone/dist/src/shell/ozone/start-ozone.sh

```diff
@@ -83,6 +83,8 @@ OZONE_JUMBO_RETCOUNTER=$?
 #---------------------------------------------------------
 # Ozone ozonemanager nodes
 OM_NODES=$("${OZONE_HOME}/bin/ozone" getconf -ozonemanagers 2>/dev/null)
+TMP_OM_NODES=$("${OZONE_HOME}/bin/ozone" getconf -ozonemanagers 2>/dev/null)
+OM_NODES=`echo $TMP_OM_NODES | grep -o '\([0-9]\{1,3\}\.\)\{3\}[0-9]\{1,3\}' | sort -d | uniq`
```

Comment:
This change seems to be unrelated to Ranger.

Reply:
Can you please remove these unrelated changes from the PR?

Instead of working on `master`, creating a new branch in your fork for each issue provides better separation of unrelated changes.

Reply:
I am very sorry that I did not read it carefully when submitting it